docs/source/user-guide/vision/tutorials/plot_custom_task_tutorial.py

```diff
@@ -25,7 +25,7 @@
 4. `Implement Custom Metric <#implement-custom-metric>`__
 """
 
-#%%
+# %%
```

Comment:
why?

Reply:
because sometimes space is pressed accidentally.